### DISCUSSION TITLE
Grafana Dashboard Export (PDF/HTML/)

### DISCUSSION THREAD

**Discussion by xfaith** [Security-Onion-Solutions/securityonion#7986] [Unsupported Versions] [answered]
Trying to see if the primary dashboard  on grafana, I was hoping to export via PDF, but does not seem available, has anyone found a way to do a readable/saveable export?

**Comment by dougburks** [comment, marked as answer]
One option might be to use something like puppeteer:
https://github.com/puppeteer/puppeteer

If you don't necessarily need the dashboards and just want the metrics, another option might be to query the influx database directly.
